src/Aspire.Hosting.Azure.AppContainers/ContainerAppExtensions.cs

```diff
@@ -166,4 +166,67 @@ public static IResourceBuilder<T> PublishAsAzureContainerAppJob<T>(this IResourc
 
         return resource;
     }
+
+    /// <summary>
+    /// Configures the specified compute resource as a manually triggered Azure Container App Job.
+    /// </summary>
+    /// <typeparam name="T">The type of the compute resource.</typeparam>
+    /// <param name="resource">The compute resource builder.</param>
+    /// <returns>The updated compute resource builder.</returns>
+    /// <remarks>
+    /// This is a convenience overload for the common case of manually triggered jobs. Manual trigger 
+    /// is the default trigger type, so this method provides a simpler API when no additional job 
+    /// configuration is needed.
+    /// 
+    /// <example>
+    /// <code>
+    /// builder.AddProject&lt;Projects.ProcessorJob&gt;("processor-job")
+    ///        .PublishAsAzureContainerAppJob(); // Manual trigger (default)
+    /// </code>
+    /// </example>
+    /// </remarks>
+    [Experimental("ASPIREAZURE002", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
+    public static IResourceBuilder<T> PublishAsAzureContainerAppJob<T>(this IResourceBuilder<T> resource)
+        where T : IComputeResource
+    {
+        ArgumentNullException.ThrowIfNull(resource);
+
+        return resource.PublishAsAzureContainerAppJob((infrastructure, job) =>
+        {
+            // Manual trigger is the default, so no additional configuration is needed
+            // This overload provides a simpler API for the common manual trigger case
+        });
+    }
+
+    /// <summary>
+    /// Configures the specified compute resource as a scheduled Azure Container App Job with the provided cron expression.
+    /// </summary>
+    /// <typeparam name="T">The type of the compute resource.</typeparam>
+    /// <param name="resource">The compute resource builder.</param>
+    /// <param name="cronExpression">The cron expression that defines the schedule for the job.</param>
+    /// <returns>The updated compute resource builder.</returns>
+    /// <remarks>
+    /// This method is a convenience wrapper around <see cref="PublishAsAzureContainerAppJob{T}(IResourceBuilder{T}, Action{AzureResourceInfrastructure, ContainerAppJob})"/>
+    /// that automatically configures the job with a schedule trigger using the specified cron expression.
+    /// 
+    /// <example>
+    /// <code>
+    /// builder.AddProject&lt;Projects.ProcessorJob&gt;("job")
+    ///        .PublishAsAzureContainerAppJob("0 0 * * *"); // Run every day at midnight
+    /// </code>
+    /// </example>
+    /// </remarks>
+    [Experimental("ASPIREAZURE002", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
+    public static IResourceBuilder<T> PublishAsAzureContainerAppJob<T>(this IResourceBuilder<T> resource, string cronExpression)
```

Comment:
I'm not sure I like that 2 overloads get you a manual schedule, and the another one gets you a "scheduled" job.

I thought the scheduled job overload should be a diferent name.